tools/stats/print_test_stats.py

```diff
@@ -805,6 +805,30 @@ def send_report_to_s3(head_report: Version2Report) -> None:
     obj.put(Body=bz2.compress(json.dumps(head_report).encode()))
 
 
+def upload_failures_to_rds(reports: Dict[str, TestFile]) -> None:
+    """
+    We have 40k+ tests, so saving every test for every commit is not very
+    feasible for PyTorch. Most of these are things we don't care about anyways,
+    so this code filters out failures and saves only those to the DB.
```

Comment:
The RDS DB is separate from the S3 ossci-metrics bucket right? So after this change, we'd be uploading to both?

Reply:
yeah, the main difference is that the failures are queryable / monitorable